config/src/main/java/org/springframework/security/config/annotation/web/AbstractRequestMatcherRegistry.java

```diff
@@ -25,11 +25,13 @@
 import org.springframework.security.web.util.matcher.AnyRequestMatcher;
 import org.springframework.security.web.util.matcher.RegexRequestMatcher;
 import org.springframework.security.web.util.matcher.RequestMatcher;
+import org.springframework.util.Assert;
```

Comment:
Can we change the copyright date to any class you modify or create in this PR to `2019` instead of `2018`?

Reply:
> Can we change the copyright date to any class you modify or create in this PR to `2019` instead of `2018`?

Fixed.